### DISCUSSION TITLE
Custom tools are not loaded if you are not logged in

### DISCUSSION THREAD

**Discussion by ikaymaz** [laravel/nova-issues#4966] [Ideas & Feature Requests]
- Laravel Version: 9.36.4
- Nova Version: 4.16.1
- PHP Version: 8.0.24
- Database Driver & Version: mysql (8.0.19)
- Operating System and Version: Ubuntu 22.04
- Browser type and version: Chrome 106.0.5249.119
- Reproduction Repository: -

### Description:
Custom tools are not loaded if you are not logged in.

In my case the standard login process is not sufficient, therefore I want to use a custom process.
We already have the possibility to customize certain routes via `config/nova.php`.
The backend part is called and executed accordingly.
When calling the corresponding Vue component via Inertia, the frontend is loaded correctly, but an error is referenced.
The problem is that custom tools aren't loaded and therefore can't be reached.

DOM logged out:
![image](https://user-images.githubusercontent.com/97731704/197761081-1d343791-e372-43f8-8848-953794c0e91c.png)

DOM logged in:
![image](https://user-images.githubusercontent.com/97731704/197761022-fbb1c042-a9a5-49ba-a0f5-4bc15c7083b3.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Use a Laravel application with Laravel Nova v4 installed
2. Create a custom tool with the console command
```
php artisan nova:tool example/test-tool
```
3. add the new tool (`TestTool`) to your `NovaServiceProvider` method called `tools`
4. Call the Nova application while logged out.
Check if the tools are present (for example, when viewing the DOM).
5. Call the Nova application while logged in.
Check if the tools are present (for example, when viewing the DOM).

**Comment by crynobone** [comment]
Current behaviour dictate all assets including Tool will only be loaded after logged-in, marking this as feature request.

**Reply by whipsterCZ** [reply]
what is exact reason? i found it as very unexpected and not documented behaviour...

**Reply by whipsterCZ** [reply]
i can access info about where the user is logged in several ways..

so give us at least option how to modify Login Form (and layout..) this is unacceptable. I Paid for not having control..
